crates/uv/tests/it/snapshots/it__ecosystem__transformers-lock-file.snap

```diff
@@ -4450,6 +4450,9 @@ wheels = [
 name = "tensorflow-macos"
 version = "2.15.1"
 source = { registry = "https://pypi.org/simple" }
+dependencies = [
+    { name = "tensorflow-macos", marker = "platform_machine == 'arm64' and platform_system == 'Darwin'" },
+]
```

Comment:
We could choose to omit these? This is a useless self-dependency. I retained them but we could omit them when converting from PubGrub resolution to our own graph.